custom_metrics/media.js

```diff
@@ -93,23 +93,51 @@ return JSON.stringify({
   })(),
   // Count all video nodes
   'num_video_nodes': document.querySelectorAll('video').length,
-  // Returns a set of video node attribute names
-  'video_nodes_attributes': (() => {
-    var allAttributes = getNodesAttributes(document.querySelectorAll('video'));
-    var filter = ['autoplay', 'autoPictureInPicture', 'buffered', 'controls',
+
+  //gets the duration of all videos loaded
+  'video_durations': (() =>{
+    var nodes = document.querySelectorAll('video');
+    var videoDuration = new Set();
```

Comment:
If two videos have the same duration, this will deduplicate them. Is that what you want, or do you want a 1:1 map of videos to durations?

Reply:
good catch - I'll fix that. :)